src/client/tile.cpp

```diff
@@ -2203,6 +2203,44 @@ video::ITexture* TextureSource::getNormalTexture(const std::string &name)
 	return NULL;
 }
 
+namespace {
+	float linear_to_srgb_component(float v)
+	{
+		if (v > 0.0031308f)
+			return 1.055f * powf(v, 1.0f / 2.4f) - 0.055f;
+		return 12.92f * v;
+	}
+	float srgb_to_linear_component(float v)
+	{
+		if (v > 0.04045f)
+			return powf((v + 0.055f) / 1.055f, 2.4f);
+		return v / 12.92f;
+	}
+
+	v3f srgb_to_linear(v3u8 col_srgb)
```

Comment:
Should this be `SColor` instead of `v3u8` (which we do not generally use)?

Reply:
It probably should. I don't remember why I have used v3u8.